public/editor/stylesheets/editor.less

```diff
@@ -651,6 +651,10 @@ body {
         background: #EEE;
       }
     }
+    
+    &:hover {
+      color: rgba(0,0,0,.5);
```

Comment:
Please change to ``color: rgba(0,0,0,.6);`` to match the 'enabled' state and we'll be good to go.